### DISCUSSION TITLE
Problem with FAPatch.m

### DISCUSSION THREAD

**Discussion by NBAlexis** [FeynCalc/feyncalc#158] [General]
When I change to my new computer, I find 

Import["https://raw.githubusercontent.com/FeynCalc/feyncalc/master/install.m"];
InstallFeynCalc[]

will hung on "Patch FeynArts"

All were OK on win10+Mathematica12+FeynCalc9.2
But will hung on win11+Mathematica13+FeynCalc9.3.1
I think this is not a bug of FeynCalc but a bug of win11 or maybe mathematica13.
However it affects the usage of FeynCalc.
BTW: In the second diaglog, I choose to use standard form, I don't known whether it is related.

I find a work around though:
1. Install FeynCalc without FeynArts
2. In Feynman/FAPatch.m, find:
ChoiceDialog["An installation of FeynArts has been found in \"" <> $FeynArtsDirectory <>
				"\". This program will now patch FeynArts to allow interoperation with FeynCalc. Continue?",
					WindowTitle->"Patch Feynarts"]
change the above to "True"
3. Download FeynArts, unpack it to AppData\Roaming\Mathematica\Applications\FeynCalc\FeynArts
4. $LoadFeynArts = True;
<< FeynCalc`;
should work now.

Thanks!

**Comment by vsht** [comment]
Thanks for this piece of information! I don't have Win 11 installed, so I can't really test it,
but I've just modified the installer to remove the additional confirmation dialog that asks
whether you really want to patch FeynArts. 

Thus, once you agree to download FeynArts, there should be no further requests regarding the
patching. Hope this helps.

BTW, if you look at `Options[InstallFeynCalc]` you will see that there are many options that allow
for installing FeynCalc without any further questions or confirmations. Setting 
`AutoInstallFeynArts` to `True` could be also helpful in your case.
